From 5e09af0ddec7fff6f683abcd0bfa8a96b61a488f Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:07:34 -0400
Subject: [usb] ftdi-elan: Coalesce string fragment

Message-id: <1410887463-20674-383-git-send-email-dzickus@redhat.com>
Patchwork-id: 95435
O-Subject: [RHEL7 PATCH 382/591] usb: ftdi-elan: Coalesce string fragment
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit a92cec2737d16c458ec04836d1271a832b56a2e6
Author: Joe Perches <joe@perches.com>
Date:   Fri Apr 4 15:16:06 2014 -0700

    usb: ftdi-elan: Coalesce string fragment

    Make it easier to grep for this.

    Neaten a trailing statement.

    Signed-off-by: Joe Perches <joe@perches.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/misc/ftdi-elan.c b/drivers/usb/misc/ftdi-elan.c
index 8cda881..10c54b3 100644
--- a/drivers/usb/misc/ftdi-elan.c
+++ b/drivers/usb/misc/ftdi-elan.c
@@ -55,8 +55,8 @@ MODULE_LICENSE("GPL");
 #define INT_MODULE_PARM(n, v) static int n = v;module_param(n, int, 0444)
 static bool distrust_firmware = 1;
 module_param(distrust_firmware, bool, 0);
-MODULE_PARM_DESC(distrust_firmware, "true to distrust firmware power/overcurren"
-   "t setup");
+MODULE_PARM_DESC(distrust_firmware,
+   "true to distrust firmware power/overcurrent setup");
 extern struct platform_driver u132_platform_driver;
 static struct workqueue_struct *status_queue;
 static struct workqueue_struct *command_queue;
@@ -590,8 +590,7 @@ static void ftdi_elan_status_work(struct work_struct *work)
     ftdi_elan_flush_targets(ftdi);
    work_delay_in_msec = 250;
   } else {
-   dev_err(&ftdi->udev->dev, "PCI device has disappeared\n"
-    );
+   dev_err(&ftdi->udev->dev, "PCI device has disappeared\n");
    ftdi_elan_cancel_targets(ftdi);
    work_delay_in_msec = 500;
    ftdi->enumerated = 0;
-- 
1.7.1